### ISSUE TITLE
Integration tests

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-broker#34]
The repo contains only unit tests. This would be good to have full integration tests. This should be coupled to development from #12.

**Comment by JulienPeloton** [comment]
This should involve scripts under `bin/`, and coverage can be easily added, e.g.:

```bash
coverage run --source=. bin/simulate_stream.py localhost:29092 ztf-stream-sim 0.2 5
```

**Comment by JulienPeloton** [comment]
Fixed in #60
